sdk/resource/builtin.go

```diff
@@ -101,3 +105,8 @@ func (defaultServiceNameDetector) Detect(ctx context.Context) (*Resource, error)
 		},
 	).Detect(ctx)
 }
+
+// Detector that does nothing. Used to disable default Detector behavior.
```

Comment:
```suggestion
// Detect does nothing. Used to disable default Detector behavior.
```

The linter will yell at you if the first word of the godoc isn't the method name.